pkg/gardenlet/operation/botanist/etcd.go

```diff
@@ -47,6 +47,15 @@ func (b *Botanist) DefaultEtcd(role string, class etcd.Class) (etcd.Interface, e
 		hvpaEnabled = features.DefaultFeatureGate.Enabled(features.HVPAForShootedSeed)
 	}
 
+	var vpaMaxAllowed corev1.ResourceList
+	if b.Config != nil &&
+		b.Config.SeedConfig != nil &&
+		b.Config.SeedConfig.Spec.Settings != nil &&
+		b.Config.SeedConfig.Spec.Settings.VerticalPodAutoscaler != nil &&
+		b.Config.SeedConfig.Spec.Settings.VerticalPodAutoscaler.MaxAllowed != nil {
+		vpaMaxAllowed = b.Config.SeedConfig.Spec.Settings.VerticalPodAutoscaler.MaxAllowed.DeepCopy()
+	}
```

Comment:
I would rather expect `MergeMinValuesIntoResourceList` to make a copy and not mutate the passed argument. Callers shouldn't care.